house_plants.sql

```diff
@@ -0,0 +1,60 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.7.41)
+# Database: house_plants
+# Generation Time: 2023-02-06 12:03:57 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table plants
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `plants`;
+
+CREATE TABLE `plants` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `colloquial_name` varchar(255) DEFAULT NULL,
+  `latin_name` varchar(255) DEFAULT NULL,
+  `size_cm` int(11) DEFAULT NULL,
```

Comment:
could change this to unsigned so it can't be a minus value

Reply:
Thanks, have made changes